### PR TITLE
[v2] ibverbs: remove unused ibv command declarations

### PR DESCRIPTION
**Pull request by ramranix**
Remove the IBV command declarations ibv_cmd_query_gid and
ibv_cmd_query_pkey as they are not implemented.

Signed-off-by: Ram Amrani <Ram.Amrani@cavium.com>

### CODE REVIEW

File: redhat/rdma-core.spec [hunk 1]
```diff
@@ -118,6 +118,7 @@ Device-specific plug-in ibverbs userspace drivers are included:
 - libnes: NetEffect RNIC
 - libocrdma: Emulex OneConnect RDMA/RoCE Device
 - librxe: A software implementation of the RoCE protocol
+- libqedr: QLogic QL4xxx RoCE HCA
```
Comment (rleon) [inline redhat/rdma-core.spec]: Please go one step further and add all missing drivers.

Thanks

Comment (jgunthorpe) [inline redhat/rdma-core.spec]: And keep the list sorted please

**Comment by ramranix** [comment]
I'll place libqedr in alphabetical location.
But I will leave the other vendors to add their own lines as I don't know
what is their supported HW.

Thanks,
Ram

**Comment by rleon** [comment]
You can find it easily by checking their kernel code and looking in their cover letters.

Please invest time and stop throwing this shit on me and Jason. We are not alone here.

**Comment by ramranix** [comment]
I appreciate your work and don't take it for granted.
I'm glad to help and take part in the effort and I'm sure that my
involvement will increase over time.
I'm not "throwing shit around" and would appreciate a lower tone.

The description we entered for qedr is not the one that appears in our
cover letter and that is why I reckoned that hns/vmw_pcrdma would like to
add it themselves.

Still, I understand that you want to lower overhead. So I'll grab what I
can from their cover letters.

Thanks,
Ram

**Pull request closed without merging**